docs/input/docs/usage/msbuild.md

```diff
@@ -258,6 +258,18 @@ For SDK-style projects, `UpdateVersionProperties` controls setting the default
 variables: `Version`, `VersionPrefix`, `VersionSuffix`, `PackageVersion`,
 `InformationalVersion`, `AssemblyVersion` and `FileVersion`.
 
+### Namespace generation
+
+You can configure GitVersion to generate the `GitVersionInformation` class in a namespace that matches the current assembly. By default this class is created in the global namespace. If `GenerateGitVersionInformationInUniqueNamespace` is set to true, the `GitVersionInfomation` class will instead be generated in a namespace matching the current project. If the property `<RootNamespace>` is set that value will be used, otherwise the name of the project file is used. 
```

Comment:
Since the namespace matches the project, I think `GenerateGitVersionInformatoinInProjectNamespace` seems more accurate. But that is also a bit long and unwieldy, so I think we can just call it `UseProjectNamespace`, no?

```suggestion
You can configure GitVersion to generate the `GitVersionInformation` class in a namespace that matches the current assembly. By default this class is created in the global namespace. If `UseProjectNamespace` is set to true, the `GitVersionInfomation` class will instead be generated in a namespace matching the current project. If the property `<RootNamespace>` is set that value will be used, otherwise the name of the project file is used. 
```

Reply:
I would be worried about such a short property name colliding with other build tasks. and a developer would not know what it was associated with just looking at a project file.
We could probably make it a tiny bit shorter by renaming it `UseProjectNamespaceForGetVersionInfomation`.
I'm not sure length is a huge issue though since it lives on a line in the project file by itself.

Reply:
> I would be worried about such a short property name colliding with other build tasks. 

Good point.


> We could probably make it a tiny bit shorter and more precise by renaming it `UseProjectNamespaceForGitVersionInfomation`. 

👍🏼